src/components/MessageInput/ActionSheetAttachment.tsx

```diff
@@ -37,13 +46,34 @@ const ActionSheetTitleText = styled.Text`
   ${({ theme }) => theme.messageInput.actionSheet.titleText.css};
 `;
 
+export type ActionSheetStyles = {
```

Comment:
another reason to drop styled components....how as a user do I know what the sdk expects when there is a mix!

Reply:
agreed